MainModule/Server/Commands/Players.lua

```diff
@@ -900,12 +900,24 @@ return function(Vargs, env)
 						local hasSafeChat = if policyInfo then
 							type(policyInfo) == "string" and policyInfo or table.find(policyInfo.AllowedExternalLinkReferences, "Discord") and "No" or "Yes"
 							else "[Redacted]"
+						
+						local mailVerif = select(2, xpcall(service.MarketplaceService.PlayerOwnsAsset, function() return "[Error]" end, service.MarketplaceService, v, 102611803))
```

Comment:
There are players who don't have this hat. They have their email verified, So there are other hats for verifying email too.

Reply:
Exactly

Reply:
you mean this old verified sign?
https://www.roblox.com/catalog/1567446
i almost forgot abt that..

Reply:
anyways should I also add the old verified sign to be also considered in the script, or just remove the HasMailVerified and only do the Player:IsVerified() thing in this PR?